utils/coreutils/Makefile

```diff
@@ -134,7 +134,7 @@ CONFIGURE_ARGS += \
 define Build/Compile
 	$(MAKE) -C $(PKG_BUILD_DIR) \
 		DESTDIR="$(PKG_INSTALL_DIR)" \
-		SHELL="/bin/bash" \
+		SHELL="/bin/sh" \
```

Comment:
can you just drop the SHELL= bit altogether?  It would seem that it was only added because someone felt it _needed_ bash?

Reply:
I just tested and that works for me as well.

Reply:
Wonder if this entire section can be removed and replaced with PKG_INSTALL.

Reply:
yeah, I was starting to look it looked pretty generic

Reply:
Already did this.